src/index/Vocabulary.h

```diff
@@ -228,6 +228,11 @@ class Vocabulary {
       vocabulary_.getUnderlyingVocabulary().resetToType(type);
     }
   }
+
+  // Determines if `getGeoInfo` can return results for this vocabulary: returns
+  // true, if the `UnderlyingVocabulary` is a `SplitGeoVocabulary` or a
+  // `PolymorphicVocabulary` holding a `SplitGeoVocabulary`.
```

Comment:
The comment can be stronger:
If true, then  getGeoInfo will return a result if and only if the index points to a valid geometry.
if false, then `getGeoInfo` will always return `nullopt`.

(Please formulate nicely, and move this declaration directly next to the getGeoInfo function, as they belong together).